### ISSUE TITLE
Output best parameters in "Summary" column in experement result.

### ISSUE THREAD

**Issue by krkhnh** [fracpete/multisearch-weka-package#6]
While I can review selected parameters from Explorer or debug output from console, I can't find it from Experiment result. This is annoying because I need to re-run the classifier in Explorer to find out the best parameters.

There is "Summary" column in experiment result(arff, xlsx, csv, ...) and I expected that I can get experiment results from there. I'm not sure, but I recall that other algorithms like GridSearch outputs selected parameters there.

For example, this "Summary" result is from Experimenter.
![clipboard01](https://cloud.githubusercontent.com/assets/5172480/12283564/c8ec43f8-b9ea-11e5-9d27-f13f1f46d0b5.png)

And this is from Excel-formated result.
![clipboard02](https://cloud.githubusercontent.com/assets/5172480/12283587/05fcceac-b9eb-11e5-8170-04ea268d6d09.png)

**Comment by krkhnh** [comment]
Ah, maybe not GridSearch, but attributeSelectedClassifier outputs selected attributes in "Summary" column.

**Comment by fracpete** [comment]
The summary string contains already the best classifier. Here is the display of an ARFF file from an example experiment run:
![experimenter_summary](https://cloud.githubusercontent.com/assets/1026180/12284807/0102ff20-ba18-11e5-9efd-20c6fe315de0.png)

**Comment by krkhnh** [comment]
Wow, I'm so clueless now... I used MultiSearch but no Summary was there. Maybe because I enclosed it with `FilteredClassifier`? I'll test it without using `FilteredClassifier` next time.

Here is the configuration string I used.
`meta.FilteredClassifier '-F \"unsupervised.attribute.Remove -R 22\" -W meta.FilteredClassifier -- -F \"unsupervised.attribute.Remove -V -R 2,3,4,6,7,8,9,11,12,13,14,15,17,18,20,21,22,23,24,25,26,27,28,29,30,35,36,37,38,39,41,43,44,54,55,56,61,62,63,67,68,69,70,71,72,76,77,78,81,83,84,85,86,87,89,90,last\" -W meta.MultiSearch -- -E MAE -search \"weka.core.setupgenerator.ListParameter -property classifier.eliminateColinearAttributes -list \\\"True False\\\"\" -search \"weka.core.setupgenerator.MathParameter -property classifier.ridge -min -10.0 -max 5.0 -step 1.0 -base 10.0 -expression pow(BASE,I)\" -sample-size 100.0 -log-file \"D:\\\\Program Files\\\\Weka-3-7\" -initial-folds 2 -subsequent-folds 10 -num-slots 6 -S 1 -W functions.LinearRegression -output-debug-info -- -S 1 -C -R 1.0E-8 -num-decimal-places 4' -4523450618538717400`
